tests/models/hf_equivalence/test_granite_vision.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import torch
+import requests
+
+from fms.models import get_model
+from fms.utils.generation import generate, pad_input_ids
+
+device = "cuda"
+torch.set_default_dtype(torch.float32)
+
+
+def _get_inputs(processor):
+    from PIL import Image
+
+    url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    inputs = "<|system|>\nA chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions.\n<|user|>\n<image>\nWhat animal is shown in this image?\n<|assistant|>\n"
+    inputs = processor(text=inputs, images=image, return_tensors="pt").to(device)
+    return inputs
+
+
+def _get_hf_model_output(model_path, inputs):
+    from transformers import LlavaNextForConditionalGeneration
+
+    model = LlavaNextForConditionalGeneration.from_pretrained(model_path).to(device)
+    model.eval()
+    with torch.no_grad():
+        output = model.generate(
+            **inputs, max_new_tokens=20, use_cache=True, do_sample=False
+        )
+    return output
+
+
+def _get_fms_model_output(model_path, inputs):
+    model = get_model(
```

Comment:
to ensure we match HF config perfectly, we should use hf_pretrained here

Reply:
Will try and enable HF config loading for llava_next, so we can use hf_pretrained. Ran into some issues in an earlier attempt, but will try again